src/components/SpruceForm/SpruceForm.stories.tsx

```diff
@@ -1,66 +1,63 @@
 import { useState } from "react";
 import { action } from "@storybook/addon-actions";
+import { StoryObj } from "@storybook/react";
 import { SpruceForm, SpruceFormContainer } from ".";
 
-export const Example1 = () => {
-  const [formState, setFormState] = useState(example1Def.formData);
-  const onChange = (d) => {
-    const { formData } = d;
-    action("Change Form State")(formData);
-    setFormState(formData);
-  };
-  return (
-    <SpruceFormContainer title="Distro Projects">
-      <SpruceForm
-        schema={example1Def.schema}
-        uiSchema={example1Def.uiSchema}
-        onChange={onChange}
-        formData={formState}
-      />
-    </SpruceFormContainer>
-  );
+export default {
```

Comment:
The stories here look pretty weird. Maybe it needs the global reset styles?
<img width="955" alt="image" src="https://user-images.githubusercontent.com/4605522/227050534-6345ae82-9644-487e-8025-6bed76eea3a5.png">

Reply:
This is actually an [existing issue](https://mciuploads.s3.amazonaws.com/spruce/spruce_ubuntu2004_storybook_5a5205cf6f3129423c382861d59c91435e6d7b88_23_03_22_22_03_15/storybook/spruce/storybook-static/index.html?path=/story/components-spruce-form--example-1) (from [this build](https://spruce.mongodb.com/version/spruce_5a5205cf6f3129423c382861d59c91435e6d7b88/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)). I added it as part of EVG-19241.